master/internal/api_experiment.go

```diff
@@ -114,7 +114,9 @@ func (a *apiServer) GetExperiment(
 		// easy deducible how long that would block. So the best we can really do is return without
 		// an error if we're in this case and log. This is a debug log because of how often the
 		// happens when polling for an experiment to end.
-		if !strings.Contains(err.Error(), actorDidNotRespond) {
+		if !strings.Contains(err.Error(), actorDidNotRespond) ||
```

Comment:
either consts for both, or remove the const I added, for consistency? either is OK with me.